requirements/edx/base.in

```diff
@@ -149,3 +149,4 @@ xblock-review                       # XBlock which displays problems from earlie
 zendesk                             # Python API for the Zendesk customer support system
 django-import-export==1.0.1
 tablib==0.9.11
+openpyxl==2.5.14
```

Comment:
empty line

Reply:
should I edit the base.in file? Or it is my mistake?